quote.cpp

```diff
@@ -0,0 +1,44 @@
+#include <string>
+
+namespace Sass {
+  using namespace std;
+
+  string unquote(const string& s)
+  {
+    if (s.empty()) return "";
+    if (s.length() == 1) {
+      if (s[0] == '"' || s[0] == '\'') return "";
+    }
+    char q;
+    if      (*s.begin() == '"'  && *s.rbegin() == '"')  q = '"';
+    else if (*s.begin() == '\'' && *s.rbegin() == '\'') q = '\'';
```

Comment:
Is it running with an implicit 'warnings as errors' setting?  Anyway, thanks for pointing it out.  When `1.0.14` <https://ci.appveyor.com/project/sass/libsass/build/1.0.14>  finally runs we'll see if I got it.